src/main/java/core/basesyntax/service/StoreReadCsvImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Store;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class StoreReadCsvImpl implements StoreReadCsv {
+
+    @Override
+    public Store read(String fileName) {
+        List<String> fruits = new ArrayList<>();
+        try {
+            fruits = Files.readAllLines(Path.of(fileName));
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find file");
+        }
+        return fruits
+                .stream()
+                .map(this::getFromCsv)
+                .findFirst()
+                .get();
+    }
+
+    private Store getFromCsv(String line) {
+        String[] fields = line.split(",");
+        Store fruit = new Store();
+        fruit.setType(Store.Operation.valueOf(fields[0]));
+        fruit.setName(fields[1]);
+        fruit.setAmount(Integer.parseInt(fields[2]));

```

Comment:
integer, string literals - to constants
Why do you create a store and name it `fruit`? :) valueOf is a static method, it doesn't require Store instance. ValueOf would work for `BALANCE`, not `b` which we have in files, so you need a separate method for that